### DISCUSSION TITLE
ZIP Quick upload

### DISCUSSION THREAD

**Discussion by showierdata9978** [modrinth/.github#179] [Feature requests]
Allow a mod developer to upload a zip, and auto fill the needed stuff based of the files in the zip

unzipping on the server for downloads 

(specifically downloaded artifacts from github)

if this is already a thing on the server make it more obvious

**Comment by Minenash** [comment]
Modrinth detects metadata from the file you upload (the jar if it's a mod/plugin). I believe it works with zips for resource packs

**Comment by showierdata9978** [comment]
im uploading a mod lol, sometimes i want to upload from mobile after
merging a PR
